app/scripts/modules/azure/serverGroup/serverGroup.transformer.js

```diff
@@ -79,10 +77,22 @@ module.exports = angular
 
       if (typeof command.customScriptsSettings !== 'undefined') {
         configuration.customScriptsSettings.commandToExecute = command.customScriptsSettings.commandToExecute;
-        if (Array.isArray(command.customScriptsSettings.fileUris)) {
-          configuration.customScriptsSettings.fileUris = command.customScriptsSettings.fileUris;
-        } else {
-          configuration.customScriptsSettings.fileUris = [command.customScriptsSettings.fileUris];
+        if (
+          typeof command.customScriptsSettings.fileUris !== 'undefined' &&
+          command.customScriptsSettings.fileUris != ''
+        ) {
+          var fileUrisTemp = command.customScriptsSettings.fileUris;
+          if (fileUrisTemp.includes(',')) {
+            configuration.customScriptsSettings.fileUris = fileUrisTemp.split(',');
+          } else if (fileUrisTemp.includes(';')) {
+            configuration.customScriptsSettings.fileUris = fileUrisTemp.split(';');
+          } else {
+            configuration.customScriptsSettings.fileUris = [fileUrisTemp];
+          }
+
+          configuration.customScriptsSettings.fileUris.forEach(function(v, index) {
+            configuration.customScriptsSettings.fileUris[index] = v.trim();
```

Comment:
It is possible the array will end up with empty strings here.  Is that a problem?

For example, if `command.customScriptsSettings.fileUris` is `" "` then `configuration.customScriptsSettings.fileUris` would equal `['']`